### PR TITLE
fix(curriculum) : removed comma and fixed sentence

### LINKED ISSUE #47686: Small fixes notified on crowdin
Just transcribing issues from crowdin.

In this line:
https://github.com/freeCodeCamp/freeCodeCamp/blob/d490c828197d68749e63ddaa5509fa1e0df20476/curriculum/challenges/english/06-quality-assurance/quality-assurance-and-testing-with-chai/assert-deep-equality-with-.deepequal-and-.notdeepequal.md?plain=1#L11
There is an extra comma before the anchor element that has to be removed.

In this line:
https://github.com/freeCodeCamp/freeCodeCamp/blob/d490c828197d68749e63ddaa5509fa1e0df20476/curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md?plain=1#L29
There is an incorrect use of parenthesis.

It should become
> Note that the day Chaos 1, 3188 YOLD the Discordian calendar is the day January 1, 2022 in the Gregorian calendar.

---

This looks like something that can be fixed by "first-time" code contributors to this repository. Here are the files that you should be looking at to work on a fix:

List of files:

1. https://github.com/freeCodeCamp/freeCodeCamp/blob/d490c828197d68749e63ddaa5509fa1e0df20476/curriculum/challenges/english/06-quality-assurance/quality-assurance-and-testing-with-chai/assert-deep-equality-with-.deepequal-and-.notdeepequal.md
2. https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md

Please make sure you read our [guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in our [chat room](https://discord.gg/PRyKn3Vbay) or our [forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing; our moderators will guide you through this.

Sometimes we may get more than one pull request. We typically accept the most quality contribution followed by the one that is made first.

Happy contributing.

### PR DESCRIPTION
**Pull request by Doraemon23**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47686

Describe your changes:
Fixed the extra comma and incorrect use of paranthesis in the sentences.

Issue ticket number and link:
Small fixes notified on crowdin #47686
https://github.com/freeCodeCamp/freeCodeCamp/issues/47686

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Holidays: `'Chaoflux', 'Discoflux', 'Confuflux', 'Bureflux', 'Afflux'`.
 
 Convert a given date from the Gregorian calendar to the Discordian calendar.
 
-Note that in the Discordian calendar (Chaos 1, 3188 YOLD) is (January 1, 2022), in the Gregorian calendar.
+Note that the day Chaos 1, 3188 YOLD the Discordian calendar is the day January 1, 2022 in the Gregorian calendar.
```
Comment (ieahleen) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md:29]:
```suggestion
Note that the day Chaos 1, 3188 YOLD in the Discordian calendar is the day January 1, 2022 in the Gregorian calendar.
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Thank you for your PR, can you please fix the sentence?

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md (hunk 1 above)
Comment (Doraemon23) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/discordian-date.md]: Hi, I have fixed the sentence as requested.

**Comment by ieahleen** [comment]
Thank you for your PR! The issue has already been fixed, so I am going to close this PR. We would welcome new contributions from you!

**Pull request closed without merging**